droidlet/tools/data_scripts/checksum_fn.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+# saves sha1sum for CHECKSUM_FOLDER at SAVE_TO_PATH 
+
+CHECKSUM_FOLDER=$1
+SAVE_TO_PATH=$2
+echo $'\nCalculating checksum for' ${CHECKSUM_FOLDER}
+# -exec calculates shasum for each file
+# cut trims the output of exec to exclude file names and finally pipes all shasums to the final shasum
+find $CHECKSUM_FOLDER -type f ! -name '*checksum*' ! -name '*MD*' \
+   -exec shasum {} \; | cut -d" " -f1 | sort -d | shasum | tr -d '-' | xargs > $SAVE_TO_PATH
+echo "Saving to ${SAVE_TO_PATH}"
+echo "Saved checksum " $(cat $SAVE_TO_PATH) $'\n'
```

Comment:
I remembered you guys talked about how this behaves differently in different OS. Is this the reason we cannot rewrite it as a python fn?

Reply:
Yes that's right, keeping this one shell script because of the checksum compute.